recipes/install_ext_services.rb

```diff
@@ -24,55 +24,15 @@
 
 case node['abiquo']['profile']
 when "monolithic", "server", "ext_services"
-    packages = %w{MariaDB-server MariaDB-client redis rabbitmq-server cronie}
-    services = %w{mysql redis rabbitmq-server crond}
+    recipes = %w{mariadb redis rabbitmq}
 when "remoteservices"
-    packages = %w{redis}
-    services = %w{redis}
+    recipes = %w{redis}
 when "monitoring"
-    packages = %w{MariaDB-server MariaDB-client}
-    services = %w{mysql}
+    recipes = %w{mariadb}
 else
-    packages = []
-    services = []
+    recipes = []
 end
 
-packages.each do |pkg|
-    package pkg do
-        action :install
-    end
+recipes.each do |recipe|
+    include_recipe "abiquo::install_#{recipe}"
 end
-
-services.each do |svc|
-    service svc do
-        action [:enable, :start]
-    end
-end
-
-passwords_match = check_db_pass()
-execute "set mysql root pass" do
-    command "/usr/bin/mysql -e 'GRANT ALL PRIVILEGES ON *.* to #{node['abiquo']['db']['user']}@\"%\" IDENTIFIED BY \"#{node['abiquo']['db']['password']}\"'"
-    action :run
-    not_if { passwords_match }
-end
-
-
-execute "create-abiquo-rabbit-user" do
-    command "rabbitmqctl add_user #{node['abiquo']['properties']['abiquo.rabbitmq.username']} #{node['abiquo']['properties']['abiquo.rabbitmq.username']}"
-    action :nothing
-    subscribes :run, "service[rabbitmq-server]"
-    not_if "rabbitmqctl list_users | egrep -q '^#{node['abiquo']['properties']['abiquo.rabbitmq.username']}.*'"
-end
-
-execute "set-abiquo-rabbit-user-administrator" do
-    command "rabbitmqctl set_user_tags #{node['abiquo']['properties']['abiquo.rabbitmq.username']} administrator"
-    action :nothing
-    subscribes :run, "execute[create-abiquo-rabbit-user]"
-end
-
-execute "set-abiquo-rabbit-user-permissions" do
-    command "rabbitmqctl set_permissions -p / #{node['abiquo']['properties']['abiquo.rabbitmq.username']} '.*' '.*' '.*'"
-    action :nothing
-    subscribes :run, "execute[create-abiquo-rabbit-user]"
-end
-
```

Comment:
I am not very sure why it is there, but since this is handled by the MariaDB cookbook, I think we could just drop it.